amoro-ams/amoro-ams-server/src/test/resources/config.yaml

```diff
@@ -0,0 +1,235 @@
+    properties:
+      spark-home: /opt/spark/                                     # Spark install home
+      master: yarn                                                # The cluster manager to connect to. See the list of https://spark.apache.org/docs/latest/submitting-applications.html#master-urls.
+      deploy-mode: cluster                                        # Spark deploy mode, client or cluster
+      export.JVM_ARGS: -Djava.security.krb5.conf=/opt/krb5.conf   # Spark launch jvm args, like kerberos config when ues kerberos
+      export.HADOOP_CONF_DIR: /etc/hadoop/conf/                   # Hadoop config dir
+      export.HADOOP_USER_NAME: hadoop                             # Hadoop user submit on yarn
+      export.SPARK_CONF_DIR: /opt/spark/conf/                     # Spark config dir
+      # spark kubernetes application properties.
+      job-uri: "local:///opt/spark/usrlib/optimizer-job.jar"      # Optimizer job main jar for kubernetes application
+      ams-optimizing-uri: thrift://ams.amoro.service.local:1261   # AMS optimizing uri
+      spark-conf.spark.dynamicAllocation.enabled: "true"          # Enabling DRA feature can make full use of computing resources
+      spark-conf.spark.shuffle.service.enabled: "false"           # If spark DRA is used on kubernetes, we should set it false
+      spark-conf.spark.dynamicAllocation.shuffleTracking.enabled: "true"                          # Enables shuffle file tracking for executors, which allows dynamic allocation without the need for an external shuffle service
+      spark-conf.spark.kubernetes.container.image: "apache/amoro-spark-optimizer:{version}"       # Optimizer image ref
+      spark-conf.spark.kubernetes.namespace: <spark-namespace>                                    # Namespace that is used within kubernetes cluster
+      spark-conf.spark.kubernetes.authenticate.driver.serviceAccountName: <spark-sa>              # Service account that is used within kubernetes cluster.
+
+
+  - name: KubernetesContainer
+    container-impl: org.apache.amoro.server.manager.KubernetesOptimizerContainer
+    properties:
+      kube-config-path: /Users/wyx/.kube/config
+#      kube-config-path: "~/.kube/config"
+      image: apache/amoro:0.7-SNAPSHOT
+      namespace: default
+      pullPolicy: "IfNotPresent"
+      podTemplate: |
+        apiVersion: apps/v1
+        kind: PodTemplate
+        metadata:
+          name: <NAME_PREFIX><resourceId>
+        template:
+          metadata:
+            labels:
+              app: <NAME_PREFIX><resourceId>
+              AmoroOptimizerGroup: <groupName>
+              AmoroResourceId: <resourceId>
+          spec:
+            containers:
+              - name: optimizer
+                image: apache/amoro:0.7-SNAPSHOT
+                imagePullPolicy: IfNotPresent
+                command: [ "sh", "-c", "<startUpArgs>" ]
+                resources:
+                  limits:
```

Comment:
We need to fill in some specific parameter values ​​for testing instead of leaving them blank